pyproject.toml

```diff
@@ -1,99 +1,92 @@
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3 :: Only",
+    "Framework :: Pydantic",
+    "Framework :: Pydantic :: 2",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Information Technology",
+    "Intended Audience :: Science/Research",
+    "Topic :: File Formats :: JSON :: JSON Schema",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Topic :: Scientific/Engineering :: GIS",
+    "Topic :: Scientific/Engineering :: Image Processing",
+    "Topic :: Scientific/Engineering :: Image Recognition",
 ]
 
+[tool.uv]
+dev-dependencies = [
+    "mypy<2.0.0,>=1.0.0",
+    "mypy-extensions<1.0.0,>=0.4.3",
+    "pre-commit<3.0.0,>=2.21.0",
+    "bandit<2.0.0,>=1.7.5",
+    "safety<3.0.0,>=2.3.4",
+    "pystac<2.0.0,>=1.10.0",
+    "pydocstyle[toml]<7.0.0,>=6.2.0",
+    "pydoclint<0.6,>=0.3",
+    "pytest<8.0.0,>=7.2.1",
+    "pytest-cov<5.0.0,>=4.1.0",
+    "pytest-mock<4.0.0,>=3.10.0",
+    "pytest-timeout<3.0.0,>=2.2.0",
+    "pytest-benchmark<5.0.0,>=4.0.0",
+    "pytest-sugar<1.0.0,>=0.9.7",
+    "pytest-click<2.0.0,>=1.1.0",
+    "pytest-pikachu<2.0.0,>=1.0.0",
+    "coverage<8.0.0,>=7.3.0",
+    "ruff<1.0.0,>=0.2.2",
+    "bump-my-version<0.27,>=0.21",
+    "types-python-dateutil>=2.9.0.20241003",
+]
 
-[tool.poetry.scripts]
-# Entry points for the package https://python-poetry.org/docs/pyproject/#scripts
-"stac-model" = "stac_model.__main__:app"
-
-
-[tool.poetry.dependencies]
-python = "^3.10"
-
-typer = {extras = ["all"], version = "^0.9.0"}
-rich = "^13.7.0"
-pydantic = "^2.6.3" # bug in post 2.3 https://github.com/pydantic/pydantic/issues/7720
-pydantic-core = "^2"
-pystac = "^1.9.0"
-shapely = "^2"
-jsonschema = "^4.21.1"
-
-[tool.poetry.group.dev.dependencies]
-mypy = "^1.0.0"
-mypy-extensions = "^0.4.3"
-pre-commit = "^2.21.0"
-bandit = "^1.7.5"
-safety = "^2.3.4"
-pystac = "^1.10.0"  # custom validator required (https://github.com/stac-utils/pystac/pull/1320)
-
-pydocstyle = {extras = ["toml"], version = "^6.2.0"}
-pydoclint = ">=0.3,<0.6"
+[project.urls]
+homepage = "https://github.com/stac-extensions/mlm/blob/main/README_STAC_MODEL.md"
+repository = "https://github.com/crim-ca/mlm-extension"
 
-pytest = "^7.2.1"
-pytest-cov = "^4.1.0"
-pytest-mock = "^3.10.0"
-pytest-timeout = "^2.2.0"
-pytest-benchmark = "^4.0.0"
-pytest-sugar = "^0.9.7"
-pytest-click = "^1.1.0"
-pytest-pikachu = "^1.0.0"
-coverage = "^7.3.0"
-ruff = "^0.2.2"
-bump-my-version = ">=0.21,<0.27"
+[project.scripts]
+stac-model = "stac_model.__main__:app"
 
 [tool.bumpversion]
 # NOTE:
 #   Although these definitions are provided in this 'stac-model' project file,
 #   they are actually intented for versioning the MLM specification itself.
-#   To version 'stac-model', use the 'poetry version' operations.
+#   To version 'stac-model', use the 'bumpversion bump' operation.
```

Comment:
`bump-my-version` is the one referenced by dev dependencies
(maybe `bumpversion` is installed as alias, but I am not sure)

Reply:
mistype, changed